central/scannerdefinitions/handler/handler.go

```diff
@@ -49,7 +49,9 @@ const (
 	// TODO(ROX-20481): Replace this URL with prod GCS bucket domain
```

Comment:
We need to remove this log line scannerdefinitions/handler: 2024/01/03 14:53:53.727186 handler.go:481: Info: Initializing updater

Reply:
That was just an extraneous log that needs to be removed. And I already removed it